apcd-cms/src/taccsite_cms/custom_app_settings.py

```diff
@@ -1,3 +1,3 @@
-CUSTOM_APPS = []
+CUSTOM_APPS = ['apps.custom_example', 'apps.submission_form']
```

Comment:
**Suggestion.** Do not enable custom app, because:

- New dev can reference enabled apps and custom app instructions.
- https://github.com/TACC/Core-CMS-Custom/pull/9 updated documentation to show this missing step.

```suggestion
CUSTOM_APPS = ['apps.submission_form']
```